client/lib/analytics/utils/refresh-country-code-cookie-gdpr.js

```diff
@@ -20,35 +20,34 @@ export default async function refreshCountryCodeCookieGdpr( signal = undefined )
 	}
 
 	if ( refreshRequest === null ) {
-		refreshRequest = requestGeoData( signal ).then( ( { countryCode, region } ) => {
-			setCountryCodeCookie( countryCode );
-			setRegionCookie( region );
-		} );
+		refreshRequest = requestGeoData( signal )
+			.then( ( { country_short, region } ) => {
```

Comment:
This is now the "raw" response from the `/geo` endpoint, therefore `country_short`.